miner/oracle_test.go

```diff
@@ -197,29 +198,3 @@ func TestOracle_OwnATXNotFound(t *testing.T) {
 	assert.Len(t, activeSet, 0)
 	assert.Len(t, proofs, 0)
 }
-
-func TestOracle_ZeroEpochWeight(t *testing.T) {
```

Comment:
An ATX with `NumUnits` 0 is not a valid ATX, it will be refused by the ATX handler.